aredis_om/model/model.py

```diff
@@ -421,6 +421,7 @@ def __init__(
         limit: Optional[int] = None,
         page_size: int = DEFAULT_PAGE_SIZE,
         sort_fields: Optional[List[str]] = None,
+        return_fields: Optional[List[str]] = None,
```

Comment:
if you're later going to assign this `[]` you can set the default here and have the validate function return `[]` then you don't need the if else branching.